academic-services/graph/reference-data-schema.md

```diff
@@ -45,10 +45,11 @@ Column # | Name | Type | Note
 6 | OfficialPage | string |
 7 | WikiPage | string |
 8 | PaperCount | long |
-9 | CitationCount | long |
-10 | Latitude | float? |
-11 | Longitude | float? |
-12 | CreatedDate | DateTime |
+9 | PaperFamilyCount | long | See [FAQ](resources-faq.md#papercount-vs-paperfamilycount)
```

Comment:
Seems odd that this is the only attribute with an FAQ link, no?

Reply:
FamilyId field in Papers.txt also has a link to FAQ.

Reply:
I agree with Rick, it feels like we should be a bit more diligent with our per-field descriptions